### ISSUE TITLE
Let `dtypes` take single value that applies to all columns AND/OR cast DateTime to String instead of raising UnsupportedColumnTypeCombinationError

### ISSUE THREAD

**Issue by deanm0000** [ToucanToco/fastexcel#256]
Sometimes I'll have a file that has some rows of disclaimers and other stuff to be skipped, followed by dates. When I try to read in the file I'll get an UnsupportedColumnTypeCombinationError: unsupported column type combination: {DateTime, String} but since I can't open the file, I don't know what the columns are to use `dtypes`. I can work around this by doing

```
column_names = [str(x) for x in range(10)]
dtypes={
               str(x):'string' for x in range(10)
           }
```
but it only works if the problematic column is in the first 10, not to mention it's annoying and overly verbose.

It'd be nice if we could just do `dtypes='string'` and then all the columns would just be a string.

**Comment by PrettyWood** [comment]
This should be resolved in v0.11.0 that just got released.
Can you try again?

**Comment by deanm0000** [comment]
@PrettyWood 

yup good to go, thanks
